examples/02-tbrom_examples/06-TBROM_FEA_static_structural_optimization.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (C) 2022 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+""".. _ref_example_TBROM_FEA_static_structural_optimization:
+
+Static structural analysis and optimization using 3D field ROM
+--------------------------------------------------------------
+
+This example shows how PyTwin can be used to perform different types of static structural analysis using 3D field ROM.
+A static structural model of a dog bone is created with Ansys Mechanical. A fixed support is applied on the right hand
+side, while a remote force is supplied on the left hand side resulting in a stress field and associated displacement.
+Non-linear structcural steel is used a representative material. A static ROM has been generated out of the original
```

Comment:
typo: structural

Reply:
Suggestion: A non-linear structural analysis with plastic material behavior (using steel) is performed.

Reply:
updated thanks